src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -3,6 +3,8 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
     release_level: BETA
+    class_names:
+      google.example.library.v1.LibraryService: LibraryClient
```

Comment:
That's what I wanted! I don't think we should use this for other renames. This rename applies to things besides classes anyway (variable names).